java/lp-service.gradle

```diff
@@ -93,18 +98,72 @@ bootJar {
     }
 }
 
-sonar {
-    properties {
-        property 'sonar.organization', "${sonarOrganization}"
-        property "sonar.host.url", "${sonarHost}"
-        property 'sonar.coverage.jacoco.xmlReportPaths', jacocoTestReport.reports.xml.destination
+test {
+    useJUnitPlatform()
+    systemProperty 'spring.profiles.active', 'test'
+    testLogging {
+        events "passed", "skipped", "failed"
     }
 }
 
-test {
+sourceSets {
+    main {
+        java.srcDirs += 'src/main/java'
+        kotlin.srcDirs += 'src/main/kotlin'
```

Comment:
This file won't load for a project which sources it without first setting a kotlin plugin first - is that definitely the behaviour you'd like?
I figure if we want to use this file for all services we may as well put the kotlin plugins used by the verification-checks-service in here, similar to the spring boot one.